### ISSUE TITLE
[Feature]: The complete requirements

### ISSUE THREAD

**Issue by Patt92** [IsaacMarovitz/Whisky#97]
### Is your feature request related to a problem?

yes, Whisky bottle doesn't start anything, even the already existing apps, like wordpad.

I'm using Ventura, imported the toolkit on launch and checked several pages, like https://www.applegamingwiki.com/wiki/Game_Porting_Toolkit

It states only the XCode 15 beta command line tools and the toolkit are required. Differing from that guide, the gaming toolkit does not contain a pkg, which isn't mentioned in the Readme here, so I think it's also not required.
DXVK options are also not visible in the configuration menu.

- Do I need the full XCode 15 Beta?
- Do I need Sonoma, I read Ventura should work just fine, but Steam
- Do I need the Homebrew installation of the gaming toolkit?
- Do I need the mentioned Crossover 22.1.1 installed?
- Any other missing dependency?

I'm using a MB Pro 16" (M1 Pro) with Ventura.

### Describe the solution you'd like

brief list of requirements

### Anything else?

maybe a check inside Whisky.

### Issue Language

- [X] Yes my issue is written in English

**Comment by vincenttsang** [comment]
Same issue on macOS 13.4
<img width="571" alt="螢幕截圖 2023-06-12 16 25 54" src="https://github.com/IsaacMarovitz/Whisky/assets/32809814/cb4f25b2-82e8-4751-a168-49de1c8da157">
I imported the tookit which is downloaded from Apple Developer Website on launch, but Whisky couldn't start anything even the iexplore.exe in the wine bottle.

**Comment by Patt92** [comment]
just found some useful information in the releases. Sonoma required for Gaming Toolkit. Doesn't explain, why nothing works, but I will give it a shot.

https://github.com/IsaacMarovitz/Whisky/releases/tag/pre-0.2.0

**Comment by IsaacMarovitz** [comment]
- Do I need the full Xcode 15 Beta?
No, you don't need Xcode or the toolit

- Do I need Sonoma, I read Ventura should work just fine, but Steam
No you don't need Sonoma

- Do I need the Homebrew installation of the gaming toolkit?
No you don't

- Do I need the mentioned Crossover 22.1.1 installed?
No you don't

- Any other missing dependency?
No

**Comment by user334** [comment]
Same issue here. Have installed it on M1 MBP with Ventura 13.4, gave it a Apple Game Porting Toolkit dmg file on the first open, created a bottle but... it won't run anything. None of the "preinstalled apps" starts, none of my external Windows exe starts (games, regular apps). I assume it still a compatibility issue since AGPT came out exclusively with Sonoma. But as changelog for the 0.2.2 states it should work on 13.1 and above so maybe we're missing something here.

**Comment by Patt92** [comment]
thanks for the reply. Maybe it's nevertheless helpful to tell in the readme, that everything is embedded, or dealt off.
Nevertheless, the best I could start was VCredist with Whisky. Even turdy games like Stray crash instantly since 0.2.3 and at least tried at 0.2.1. Maybe I have to try out more games.

**Comment by user334** [comment]
@Patt92 please ping here if you'd manage it to work on Ventura.

**Comment by Patt92** [comment]
actually I switched over to Sonoma and use the Homebrew version. It's straight forward and with using aliases it's easy to use. https://www.applegamingwiki.com/wiki/Game_Porting_Toolkit
Ventura should be working too.

I added to the .zprofile
eval "$(/usr/local/bin/brew shellenv)"
alias gameon='gameportingtoolkit-no-hud ~/gametool-brew $@'
alias steam="gameon 'C:\Program Files (x86)/Steam/steam.exe'"

now I can type steam in the terminal and it will launch. With "gameon control" I can access the wine control panel and launch everything from software or run apps, of course it works with "gameon Path/to/exe".

Whisky would be smarter, but it's somehow not working for a few people, the logs are not really helpful and the dev most likely is not affected of that issue, which why he won't fix it with only the log in near time. There are many open issues with that issues.
